app/main/controllers/activity/RTMediaBuddyPressActivity.php

```diff
@@ -1360,4 +1365,52 @@ public function rtm_check_privacy_for_comments( $comment_id, $r ) {
 		$rtm_activity_model = new RTMediaActivityModel();
 		$rtm_activity_model->set_privacy( $comment_id, $user_id, $privacy_id );
 	}
+
+	/**
+	 * Filter contents to display in edit box if media is present
+	 *
+	 * @param string $content content to be filtered.
+	 */
+	public function rtm_edit_activity_filter( $content ) {
+		if ( strpos( $content, '<span>' ) !== false ) {
+			$content = explode( '<span>', $content );
+			$result  = explode( '</span>', $content[1] );
```

Comment:
Please check $content[1] exist or not. Incase no span tag found inside content then it will through error. Same for $result